docs/pages/develop/development-builds/create-a-build.mdx

```diff
@@ -41,141 +80,150 @@ Apps that don't use [Continuous Native Generation](/workflow/continuous-native-g
+Read more about [local app development](/guides/local-app-development/#local-builds-with-expo-dev-client).
 
 </Tab>
+</Tabs>
+
+</Step>
 
-<Tab>
+<Step label="3">
+## Install the app
 
-> **warning** Apple Developer membership is required to create and install a development build on an iOS device.
+Now you'll need to install the native app on your device, emulator or simulator.
 
-To register any iOS device you'd like to develop onto your [ad hoc provisioning profile](/build/internal-distribution/#22-configure-app-signing-credentials-for-ios), run the following command:
+### When building on EAS
 
-<Terminal cmd={['$ eas device:create']} />
+If created your Development Build on EAS, the CLI will prompt you to install the app the build is finished. You can also install previous builds from the expo.dev dashboard or using [Expo Orbit](https://expo.dev/orbit).
 
-After registering your iOS device, you can create the development build by running the command:
+### When building locally using EAS CLI
 
-<Terminal cmd={['$ eas build --profile development --platform ios']} />
+When building locally the output of the build will be an archive. You may drag and drop this on your iOS Simulator / Android Emulator to install it, or use [Expo Orbit](https://expo.dev/orbit) to install a build from your local machine onto any platform.
 
-> Devices running iOS 16 and above require enabling a special OS-level Developer Mode to install development builds. If you don't have this setting enabled or are installing your first development build on your device, see [iOS Developer Mode](/guides/ios-developer-mode/) to enable it.
+### When building locally without EAS CLI
 
-After the build is complete, you can download it on your iOS device by scanning the QR code from the device's camera from the Expo CLI. The QR code is provided when the `eas build` command has finished running.
+Local builds will get installed automatically once the build finishes.
 
-You can also find this QR code on the build page in the [Expo dashboard](https://expo.dev/accounts/[account]/projects/[project]/builds). Click the **Install** button and scan the QR code using the system's camera.
+</Step>
 
-</Tab>
+<Step label="3">
+## Start the bundler
+
+The Development Client builtin step **2** is the native side of your app (basically your own version of Expo Go!). So, to continue developing you'll want to also start the JavaScript bundler.
+
+Depending on how you built the app, this may already be running, but if you close the process for any reason, there is no need to rebuild your development client. Simply restart the JavaScript bundler with:
+
+<Terminal cmd={['npx expo start']} />
+
+This is the same command as you would have used with Expo Go. It detects whether your project has `expo-dev-client` installed in which case it will default to targeting your development build instead of Expo Go.
```

Comment:
```suggestion
This is the same command you would have used with Expo Go. It detects whether your project has `expo-dev-client` installed, in which case it will default to targeting your development build instead of Expo Go.
```